app/helpers/application_helper.rb

```diff
@@ -130,6 +130,11 @@ def badge_for_count(count)
     tag.span count, class: "badge badge-pill badge-success"
   end
 
+  def yes_no_badge(bool)
+    return tag.span("Yes", class: "badge badge-pill badge-success") if bool
+    tag.span "No", class: "badge badge-pill badge-light"
+  end
+
```

Comment:
`badge_for_boolean` might be a more consistent name given some of the other badge helpers.